### PR TITLE
feat(browser-starish): resource module ui changes

### PR DESCRIPTION
**Pull request by DominikB2014**
Bunch of changes to work towards addressing UI differences in https://github.com/getsentry/sentry/issues/58723
Didn't want to do it all in one PR to make the review more managable

1. add selector for JS/CSS and Images
image
- This still needs to be hooked up to a image view.

- This PR makes the change so that the  JS/css filters on `resource.script` and `resource.css`

2. Add resource size to the resource summary page
image
- For some reason these are all 0's tho, need to look into that.

3. Remove the `uncompressed column` as this was not possible to obtain using a timeseries.

4. Add a render blocking resource toggle switch

### CODE REVIEW

**Comment by AbhiPrasad** [comment]
On a wider screen the tab jumps to the opposite side of the page.

image

**Comment by DominikB2014** [comment]
> On a wider screen the tab jumps to the opposite side of the page.
> 
> image

Yikes, thanks for bringing that up @AbhiPrasad ! I'll look into it

**Comment by DominikB2014** [comment]
@AbhiPrasad should be good now, thanks for bringing it up!

**Comment by AbhiPrasad** [comment]
Should we still render the `render blocking` column? If we remove it we can get rid of the duplicate columns

**Comment by DominikB2014** [comment]
> Should we still render the `render blocking` column? If we remove it we can get rid of the duplicate columns

Yeah that's a good point now that we have a filter for it as well. I can make that change

**Comment by DominikB2014** [comment]
@AbhiPrasad Removed the column!
image

File: static/app/views/performance/browser/resources/index.tsx [hunk 1]
```diff
@@ -75,8 +105,14 @@ function ResourcesLandingPage() {
             <DomainSelector value={filters[SPAN_DOMAIN] || ''} />
             <ResourceTypeSelector value={filters[RESOURCE_TYPE] || ''} />
             <PageSelector value={filters[TRANSACTION] || ''} />
+            <SwitchContainer>
+              <SwitchButton
+                isActive={filters[RESOURCE_RENDER_BLOCKING_STATUS] === 'blocking'}
+                toggle={handleBlockingToggle}
+              />
+              Render Blocking
```
Comment (AbhiPrasad) [inline static/app/views/performance/browser/resources/index.tsx:113]: needs to be translated.

I'm still 50/50 on the toggle UX, but lets ship this now and get design/product help later.

**Review by AbhiPrasad — APPROVED** [review]

File: static/app/views/performance/browser/resources/index.tsx (hunk 1 above)
Comment (DominikB2014) [reply static/app/views/performance/browser/resources/index.tsx:113]: Yeah agreed!

**Pull request merged**